docs/topics/quickstart.md

```diff
@@ -35,8 +35,17 @@ All pre-configured flavors can be found on the [release page](https://github.com
 
 ## Hetzner Dedicated / Bare Metal Server
 
-If you want to create a cluster with bare metal servers, you need to additionally set up the robot credentials in the preparation step. As described in the [reference](/docs/reference/hetzner-bare-metal-machine-template.md), you need to manually buy bare metal servers before-hand.
-Then you need to create a `HetznerBareMetalHost` object for each bare metal server that you bought and specify its server ID in the specs. If you already know the WWN of the storage device you want to choose for booting, then specify it in `rootDeviceHints`. If not, then you can wait for the host to start provisioning. It will show you an error that you did not specify the WWN. If you reach this point, have a look at the status of `HetznerBareMetalHost`. There you find `hardwareDetails`, in which you can see a list of all relevant storage devices as well as their properties. You can just copy+paste the WWN of your favorite storage device into `rootDeviceHints`.  ....
+If you want to create a cluster with bare metal servers, you need to additionally set up the robot credentials in the preparation step. As described in the [reference](/docs/reference/hetzner-bare-metal-machine-template.md), you need to manually buy bare metal servers before-hand. To use bare metal servers for your deployment, you should choose one of the following flavors:
+
+| Flavor                                       | What it does                                                                                        |
+| -------------------------------------------- | --------------------------------------------------------------------------------------------------- |
+| hetzner-baremetal-control-planes-remediation | Uses bare metal servers for the control plane nodes - with self remediation                         |
```

Comment:
One minor thing, because it's technically not completly correct. This flavor is an example for the custom remediation feature which tries in case of an unknown or if ready=false to reboot the machine first. Often a reboot will already solve the issue with the machine and is much more quicker then to remove the node and to re-provision it. 
The flavor hetzner-baremetal-control-planes has a remediation (MachineHealthCheck). In case of an failure the machine will be automatically remediated. This means it will be "deleted" or un-provisioned. And a new machine will be created (Most of the time you only have the number of baremetal machines available that you actually use so the same machine will be reused).

Reply:
Thanks for the feedback and your patience. This is actually my very first PR ever... so here's to a many more! ;)